lib/rabbitmq/types/email_message.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+type EmailMessage struct {
+	To      string
+	Subject string
+	Body    string
```

Comment:
did u forget to add json struct tags? otherwise, this can't marshal/unmarshal.
```go
type EmailMessage struct {
 	To	string `json:"to"`
 	Subject	string `json:"subject"`
 	Body	string `json:"body"`
}
```